### PR TITLE
 Create roadmap-item issue template

### PR DESCRIPTION
**Pull request by vincent6767**
# Description
The context of the roadmap item can be read here: https://github.com/o3de/sig-release/issues/79.

# Action Needed
Since the template automatically assigns **needs-triage** and **kind/roadmap** to the roadmap item issue, please create this label in your SIG repo if it does not exist already once you merge the PR.

### needs-triage
Taken from [O3DE label](https://github.com/o3de/o3de/labels?q=needs-triage)
- label name: needs-triage
- description: Indicates an issue or PR lacks a triage/foo label and requires one

### kind/roadmap
- label name: **kind/roadmap**
- description: **Categorizes an issue that goes in the O3DE Public Roadmap**

Signed-off-by: Vincent <vincentbiasa6767@gmail.com>

### CODE REVIEW

**Comment by vincent6767** [comment]
Hi @o3de/sig-core-reviewers , please help to review!

**Comment by lumberyard-employee-dm** [comment]
> Hi @o3de/sig-core-reviewers , please help to review!

Hello @vincent6767 I have went ahead and create the `needs-triage` and `kind/roadmap` item labels in the sig-core repo.
I also added the `triage/accepted` label since we are missing that as well.

**Review by lumberyard-employee-dm — APPROVED** [review]
Thanks for creating the issue template

**Comment by vincent6767** [comment]
Thanks for creating the `triage/accepted`! Can I merge it or should I wait for another approval? @lumberyard-employee-dm

**Comment by lumberyard-employee-dm** [comment]
> Thanks for creating the `triage/accepted`! Can I merge it or should I wait for another approval? @lumberyard-employee-dm

Can you wait until @amzn-pratikpa reviews the template?

**Comment by vincent6767** [comment]
Yes sure. I'll wait for the @amzn-pratikpa's review.

**Review by amzn-pratikpa — APPROVED** [review]

**Comment by vincent6767** [comment]
Thanks! @amzn-pratikpa I'll merge the PR.

**Pull request merged**
